receiver/dummyreceiver/documentation.md

```diff
@@ -0,0 +1,33 @@
+[comment]: <> (Code generated by mdatagen. DO NOT EDIT.)
+
+# dummy
+
+## Default Metrics
+
+The following metrics are emitted by default. Each of them can be disabled by applying the following configuration:
+
+```yaml
+metrics:
+  <metric_name>:
+    enabled: false
+```
+
+### dummy
+
+a dummy gauge
+
+| Unit | Metric Type | Value Type |
+| ---- | ----------- | ---------- |
+| { event } | Gauge | Int |
```

Comment:
What will be the metric name? Or is that dynamic dependent on what metric_name you define in the config?

Reply:
the metric will be called `dummy`

Reply:
ahh, puuh, that's why that headline is so strange, now I get it...